R/doeFull.R

```diff
@@ -16,92 +16,99 @@
 #
 
 #' @export
-doeFull <- function(jaspResults, dataset, options, ...){
+doeFull <- function(jaspResults, dataset, options, ...) {
+  if (options[["setSeed"]]) {
+    set.seed(options[["seed"]])
+  }
 
   .doeFullSummaryDesign(options, jaspResults, position = 1)
 
   .doeFullShowDesign(options, jaspResults, position = 2)
-
 }
 
-.doeFullSummaryDesign <- function(options, jaspResults, position){
-
-  if(is.null(jaspResults[["selectedDesignFull"]])){
+.doeFullSummaryDesign <- function(options, jaspResults, position) {
+  if (is.null(jaspResults[["selectedDesignFull"]])) {
     table <- createJaspTable(gettext("Selected Design"))
     table$position <- position
-    table$dependOn(options = c("numberOfFactors",
-                               "factors",
-                               "fullCornerReplicates"))
-
-    table$addColumnInfo(name = 'type', title = "Design", type = 'string')
-    table$addColumnInfo(name = 'factors', title = gettext("Factors"), type = 'integer')
-    table$addColumnInfo(name = 'runs', title = gettext("Runs"), type = 'integer')
-    table$addColumnInfo(name = 'resolution', title = gettext("Resolution"), type = 'string')
-    table$addColumnInfo(name = 'replicates', title = gettext("Replicates"), type = 'integer')
+    table$dependOn(options = c(
+      "numberOfFactors",
+      "factors",
+      "fullCornerReplicates"
+    ))
+
+    table$addColumnInfo(name = "type", title = "Design", type = "string")
```

Comment:
```suggestion
    table$addColumnInfo(name = "type", title = gettext("Design"), type = "string")
```

Reply:
This file `doeFull.R` is not used anymore actually, both analyses are merged in `doeFactorial.R` now. These changes are from reformatting old code a while ago I think